go.mod

```diff
@@ -17,13 +17,13 @@ require (
 	golang.org/x/sys v0.7.0
```

Comment:
Why bump k8s is a breaking change for CR and not ✨ ?

Reply:
It used to be but #2223 addressed the breakage properly so I can remove ✨

Reply:
Sorry, are the end-user affected by those changes? Will they need to change their code when they bump a new release within this? If not, then IMO it seems that the right emoji is  `:sparkles:`

Reply:
changed